mtools/util/logevent.py

```diff
@@ -205,6 +207,18 @@ def duration(self):
 
         return self._duration
 
+    @property
+    def cursor(self):
+        """Pull the cursor information if available (lazy)."""
+        line_str = self.line_str
+        # SERVER-28604 Checking reaped cursor information
+        if "Cursor id" in self.line_str:
+            self._cursorid = str(line_str[line_str.find("Cursor id") + len("Cursor id"): - 55])
```

Comment:
see the note in a different review about searching with regex and groups, this makes this kind of ops more easy, we can talk offline if you have questions